omop-fhir-translator/README.md

```diff
@@ -45,6 +45,8 @@ to get person.json files that are valid you need to run
 data_cleaner.py and take the outputed file and replace it with data/person.json
 
 ## Convert OMOP -> FHIR
+Run lathe plan commands from root of project
+
```

Comment:
This isn't working for me.

### separate directory just for this PR
```$ pwd
/Users/walsbr/bmeg/iceberg-schema-tools-PR-8/iceberg-schema-tools
(venv) RNB10644:iceberg-schema-tools walsbr$ ls -l
total 64
-rw-r--r--   1 walsbr  1971611142   981 Jun 12 12:12 README.md
-rw-r--r--   1 walsbr  1971611142  5737 Jun 12 12:12 config.yaml
drwxr-xr-x   2 walsbr  1971611142    64 Jun 12 12:12 iceberg
drwxr-xr-x  14 walsbr  1971611142   448 Jun 12 13:00 omop-fhir-translator
-rwxr-xr-x   1 walsbr  1971611142    30 Jun 12 12:12 pre-commit-tests.sh
-rw-r--r--   1 walsbr  1971611142   121 Jun 12 12:12 requirements-dev.txt
-rw-r--r--   1 walsbr  1971611142   219 Jun 12 12:12 requirements.txt
drwxr-xr-x   4 walsbr  1971611142   128 Jun 12 12:12 resources
-rw-r--r--   1 walsbr  1971611142  5660 Jun 12 12:12 setup.py
drwxr-xr-x   5 walsbr  1971611142   160 Jun 12 12:12 tests
drwxr-xr-x   5 walsbr  1971611142   160 Jun 12 12:12 tools
drwxr-xr-x   6 walsbr  1971611142   192 Jun 12 12:22 venv
(venv) RNB10644:iceberg-schema-tools walsbr$ git log --oneline | head -1
e0c64d9 bug fixes
```
### Running from root of project gives this error

```
(venv) RNB10644:iceberg-schema-tools walsbr$ lathe plan plan_omop_to_fhir.yaml
log: Error: failed to read config at path /Users/walsbr/bmeg/iceberg-schema-tools-PR-8/iceberg-schema-tools/plan_omop_to_fhir.yaml:
open /Users/walsbr/bmeg/iceberg-schema-tools-PR-8/iceberg-schema-tools/plan_omop_to_fhir.yaml: no such file or directory
Error: failed to read config at path /Users/walsbr/bmeg/iceberg-schema-tools-PR-8/iceberg-schema-tools/plan_omop_to_fhir.yaml:
open /Users/walsbr/bmeg/iceberg-schema-tools-PR-8/iceberg-schema-tools/plan_omop_to_fhir.yaml: no such file or directory```


### Contents

```ls -l omop-fhir-translator/
total 40
-rw-r--r--   1 walsbr  1971611142  1845 Jun 12 12:57 README.md
-rw-r--r--   1 walsbr  1971611142  4083 Jun 12 14:29 Snakefile
drwxr-xr-x@ 14 walsbr  1971611142   448 Jun 12 14:19 data_used
drwxr-xr-x   4 walsbr  1971611142   128 Jun 12 12:13 extract
drwxr-xr-x  13 walsbr  1971611142   416 Jun 12 12:13 fhir_to_omop
drwxr-xr-x   7 walsbr  1971611142   224 Jun 12 13:00 iceberg
drwxr-xr-x  15 walsbr  1971611142   480 Jun 12 14:19 omop_to_fhir
drwxr-xr-x   2 walsbr  1971611142    64 Jun 12 12:59 output
-rw-r--r--   1 walsbr  1971611142   103 Jun 12 12:13 plan_fhir_to_omop.yaml
-rw-r--r--   1 walsbr  1971611142   101 Jun 12 12:13 plan_omop_to_fhir.yaml
-rw-r--r--   1 walsbr  1971611142    63 Jun 12 12:13 requirements.txt```


```$ ls -l omop-fhir-translator/data_used/
total 5960296
-rw-rw-r--@ 1 walsbr  1971611142  1159536475 Jun 12  2023 Eunomia_concept.json
-rw-rw-r--@ 1 walsbr  1971611142    36582988 Jun 12  2023 Eunomia_condition_occurrence.json
-rw-rw-r--@ 1 walsbr  1971611142      836016 Jun 12  2023 Eunomia_observation.json
-rw-rw-r--@ 1 walsbr  1971611142  1751431603 Jun 12  2023 MASTER_CONCEPT_TABLE_MODIFIED.json
-rw-rw-r--@ 1 walsbr  1971611142     1172849 Jun 12  2023 mimic-iv_measurement.json
-rw-rw-r--@ 1 walsbr  1971611142       47233 Jun 12  2023 mimic-iv_person.json
-rw-rw-r--@ 1 walsbr  1971611142       83775 Jun 12  2023 mimic-iv_specimen.json
-rw-rw-r--@ 1 walsbr  1971611142      537585 Jun 12  2023 mimic-iv_visit_occurrence.json
-rw-rw-r--@ 1 walsbr  1971611142      447978 Jun 12  2023 redivis_location.json
-rw-rw-r--@ 1 walsbr  1971611142    45457114 Jun 12  2023 redivis_person.json
-rw-r--r--@ 1 walsbr  1971611142    40023571 Jun 12 14:03 redivis_procedure_occurrence.json
```